py/desispec/coaddition.py

```diff
@@ -236,13 +239,16 @@ def coadd_fibermap(fibermap, onetile=False):
     else :
          raise KeyError("no FIBERSTATUS nor COADD_FIBERSTATUS column in fibermap")
 
+    # SJ: for verbose testing (commenting out)
+    #print(f"Will loop over N={len(targets)} targets")
```

Comment:
Please remove these debugging prints before we merge.

Reply:
Done.